apps/ledger/src/schema/repositories/schema.repository.ts

```diff
@@ -176,6 +179,24 @@ export class SchemaRepository {
     }
   }
 
+  async updateSchemaDetails(did: string): Promise<Prisma.BatchPayload> {
```

Comment:
We can maybe rename this function to something like `archieveSchema` or `archieveSchemaByDid`.
What do you think?

Reply:
Yes, correct. I think archiveSchema is a more descriptive function name.